### PR TITLE
translate: scope/move

### LINKED ISSUE #66: scope/move
1. [mut.md](https://github.com/nebula-labs/rust-by-example-vn/blob/main/src/scope/move/mut.md)
2. [partial_move.md](https://github.com/nebula-labs/rust-by-example-vn/blob/main/src/scope/move/partial_move.md)

### PR DESCRIPTION
**Pull request by tuannkhoi**
Close #66

1. [mut.md](https://github.com/nebula-labs/rust-by-example-vn/blob/main/src/scope/move/mut.md)
2. [partial_move.md](https://github.com/nebula-labs/rust-by-example-vn/blob/main/src/scope/move/partial_move.md)

### CODE REVIEW

File: src/scope/move/mut.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+
+Tính biến đổi của dữ liệu có thể thay đổi khi quyền sở hữu được chuyển giao.
+
+```rust,editable
+fn main() {
+    let immutable_box = Box::new(5u32);
+
+    println!("immutable_box chứa {}", immutable_box);
+
+    // Lỗi biến đổi
```
Comment (HongThaiPham) [inline src/scope/move/mut.md]:
```suggestion
    // Lỗi biến đổi (không thể thay đổi giá trị biến immutable)
```

File: src/scope/move/mut.md [hunk 2]
```diff
@@ -1 +1,24 @@
...
+    //*immutable_box = 4;
+
+    // *Di chuyển* hộp, thay đổi quyền sở hữu (và khả năng biến đổi)
```
Comment (HongThaiPham) [inline src/scope/move/mut.md]:
```suggestion
    // *Move* hộp, sẽ thay đổi quyền sở hữu (và khả năng biến đổi) của nó
```

File: src/scope/move/mut.md [hunk 3]
```diff
@@ -1 +1,24 @@
...
+    let mut mutable_box = immutable_box;
+
+    println!("mutable_box chứa {}", mutable_box);
+
+    // Sửa đổi nội dung của hộp
```
Comment (HongThaiPham) [inline src/scope/move/mut.md]:
```suggestion
    // Sửa đổi nội dung của hộp (hợp lệ vì biến là mutable)
```

File: src/scope/move/partial_move.md [hunk 4]
```diff
@@ -1 +1,47 @@
 # Partial moves
+
+Trong quá trình [destructuring] (giải cấu trúc) của một biến, cả hai mẫu ràng buộc
+`theo cách di chuyển` và `theo tham chiếu` có thể được sử dụng đồng thời.
+Việc làm này sẽ dẫn đến việc _di chuyển một phần_ của biến, điều này có nghĩa
```
Comment (HongThaiPham) [inline src/scope/move/partial_move.md]:
```suggestion
Điều này sẽ dẫn đến việc _di chuyển một phần_ của biến, có nghĩa
```

File: src/scope/move/partial_move.md [hunk 5]
```diff
@@ -1 +1,47 @@
...
+là một số phần của biến sẽ được di chuyển trong khi các phần khác vẫn ở nguyên
```
Comment (HongThaiPham) [inline src/scope/move/partial_move.md]:
```suggestion
là một số phần của biến sẽ được di chuyển trong khi các phần khác vẫn được giữ nguyên
```

File: src/scope/move/partial_move.md [hunk 6]
```diff
@@ -1 +1,47 @@
...
+chỗ. Trong trường hợp như vậy, biến cha không thể được sử dụng sau đó như một
```
Comment (HongThaiPham) [inline src/scope/move/partial_move.md]:
```suggestion
.Trong trường hợp đó, biến cha không thể được sử dụng sau đó như một
```

File: src/scope/move/partial_move.md [hunk 7]
```diff
@@ -1 +1,47 @@
...
+toàn thể, tuy nhiên các phần chỉ được tham chiếu (và không được di chuyển) vẫn có thể được sử dụng.
```
Comment (HongThaiPham) [inline src/scope/move/partial_move.md]:
```suggestion
thể duy nhất, tuy nhiên các phần chỉ được tham chiếu (và không được di chuyển) vẫn có thể được sử dụng.
```

**Comment by tuannkhoi** [comment]
applied all suggestions, cảm ơn anh @HongThaiPham

**Review by HongThaiPham — APPROVED** [review]

**Pull request merged**